### PR TITLE
Concrete Compressive Strength Prediction

### LINKED ISSUE #380: Concrete Strength Detection Using DL
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : </br>
Concrete Strength Detection Using DL
:red_circle: **Aim** : </br>
This data set aims to provide a start up for the ones who just started off with deep learning and act as a benchmark.
:red_circle: **Dataset** : </br>
https://www.kaggle.com/datasets/prathamtripathi/regression-with-neural-networking/data
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.
Comparing XGboost vs LightGBM performance on this task
**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by aindree-2005**
# Pull Request for DL-Simplified 💡

## Issue Title : Concrete Compressive Strength 
- **Info about the related issue (Aim of the project)** : 
Concrete Compressive Strength  Prediction with Multiple Models
- **Name:** Aindree
- **GitHub ID:** aindree2005
- **Email ID:** aindree2005@gmail.com
- **Idenitfy yourself: `Codepeak Participant`

closes #380 
### Describe the add-ons or changes you've made 📃
Comparison between LGBMRegressor, and Neural Network
Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

Tested on Kaggle

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Hi @aindree-2005 nicely represented. Few changes are required. Please update the `README.md` file as per the given template, https://github.com/abhisheks008/DL-Simplified/blob/main/.github/readme_template.md

**Comment by aindree-2005** [comment]
> Hi @aindree-2005 nicely represented. Few changes are required. Please update the `README.md` file as per the given template, https://github.com/abhisheks008/DL-Simplified/blob/main/.github/readme_template.md

@abhisheks008 fixed it now! Changed Readme in Models folder to include this data

**Review by abhisheks008 — APPROVED** [review]
Looks good to me. Your PR is approved and ready to be merged.
@aindree-2005
